### PR TITLE
carla: 2.1.1 -> 2.2.0-RC2

### PR DESCRIPTION
**Pull request by suhr**
###### Motivation for this change

Though this is a release candidate, it has an essential feature: VST3 support.

###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/applications/audio/carla/default.nix [hunk 1]
```diff
@@ -1,9 +1,11 @@
-{ stdenv, fetchFromGitHub, alsaLib, file, fluidsynth, ffmpeg_3, jack2,
-  liblo, libpulseaudio, libsndfile, pkgconfig, python3Packages,
-  which, withFrontend ? true,
-  withQt ? true, qtbase ? null, wrapQtAppsHook ? null,
-  withGtk2 ? true, gtk2 ? null,
-  withGtk3 ? true, gtk3 ? null }:
+{ stdenv, fetchFromGitHub, alsaLib, file, fluidsynth, ffmpeg_3, jack2
```
Comment (doronbehar) [inline pkgs/applications/audio/carla/default.nix:1]:
```suggestion
{ stdenv, fetchFromGitHub, alsaLib, file, fluidsynth, ffmpeg, jack2
```

**Review by doronbehar — COMMENTED** [review]
I won't merge this until it's 2.2.0 is fully released, but it's OK to get ready for it and enjoying it from your branch of Nixpkgs :).

**Comment by doronbehar** [comment]
Also note the aarch builds fail.

**Comment by SuperSandro2000** [comment]
@suhr please resolve the merge conflict.

**Comment by suhr** [comment]
Nixpkgs already contains Carla 2.2.0.

**Pull request closed without merging**